overlord/devicestate/devicestate_bootconfig_test.go

```diff
@@ -49,7 +49,8 @@ type deviceMgrBootconfigSuite struct {
 var _ = Suite(&deviceMgrBootconfigSuite{})
 
 func (s *deviceMgrBootconfigSuite) SetUpTest(c *C) {
-	s.deviceMgrBaseSuite.SetUpTest(c)
+	classic := false
+	s.deviceMgrBaseSuite.setupBaseTest(c, classic)
```

Comment:
It seems to me that all these changes are not needed, since you also updated the manager base suite to call `setupBaseTest(c, classic=false)`, are they?

Reply:
deviceMgrSuite (not to be confused with deviceMgrBaseSuite) uses classic=false, yes. deviceMgrBaseSuite is the base of deviceMgrSuite.

deviceMgrBaseSuite is the foundation many other test suites, in particular preseedingClassicSuite which passes classic=true down to deviceMgrBaseSuite. There will be preseedingUC20Suite which will use classic=false.

I hope this makes sense :)

Reply:
Ah, too many Mgr*Suites in there! :-)